### ISSUE TITLE
Word delete doesn't seem to delete, just moves cursor

### ISSUE THREAD

**Issue by iptq** [matrix-org/matrix-rich-text-editor#321]
steps:

1. Type `asdf`
   ```rs
   let mut model = cm("{asdf}|");
   ```
2. Ctrl-backspace, or whatever the equivalent is on Mac
   ```rs
   let mut model = cm("|asdf");
   ```
   This is not the expected behavior! It should be `cm("|")`, or `cm("")`. The problem becomes apparent during the next step
3. Start typing anything. `asdf` comes back

(Tested on the web interface)

**Comment by florianduros** [comment]
The deleteWordBackward and deleteWordForward is currently under development, it should be available soon.

**Comment by alunturner** [comment]
@iptq apolgies this is slightly delayed, but this issue was fixed in v0.10.0 of the library. Marking this issue as resolved/closed to tidy up some of the admin here.
